README.md

```diff
@@ -1,25 +1,25 @@
 [![CircleCI](https://circleci.com/gh/giantswarm/awscnfm.svg?style=shield)](https://circleci.com/gh/giantswarm/awscnfm)
 [![GoDoc](https://godoc.org/github.com/giantswarm/awscnfm?status.svg)](https://pkg.go.dev/github.com/giantswarm/awscnfm?tab=overview)
 
-
-
 # awscnfm
 
-Conformance test tool for AWS.
+Conformance test tool for Kubernetes on AWS by Giant Swarm.
+
+## Basic concepts
 
+Tests are organized based on **cluster scopes** containing multiple **actions**. Each cluster scope represents a distinct cluster configuration. The actions take care of things like creating the cluster, checking that it functions as expected, and deleting it.
 
+For example, there could be one cluster scope for a cluster with one node pool and HA masters, while another cluster scope would be for single master and no node pools.
 
-### Docs
+You can either [execute](/giantswarm/awscnfm/blob/master/docs/using.md) the entire cluster scope or a single action.
 
-* [Completion](/docs/completion.md)
-* [Environment](/docs/environment.md)
-* [Generation](/docs/generation.md)
-* [Introduction](/docs/introduction.md)
-* [Structure](/docs/structure.md)
+The `awscnfm` major and minor **version** used for testing is supposed to match the major and minor version of the release used by the tenant cluster(s) to test.
```

Comment:
One thing I was wondering is how we then know which release to use in case the versions are not equal on the patch level. We then have to be more smart about finding the right release based on major and minor and we just do not do that right now. It is a matter of time until it breaks.

Reply:
Always use the newest patch version within the same minor.